app/views/catalog/_index_split_default.html.erb

```diff
@@ -16,7 +16,17 @@
   </div>
   <div class="col-12 col-lg-3">
     <div class="thumbnail">
-      <%= gbl_thumbnail_img(document) %>
+      <% if document.sidecar.image.attached? %>
+        <% if document.sidecar.image.variable? %>
+          <%= image_tag document.sidecar.image.variant(resize: "200"), {class: 'mr-3'} %>
```

Comment:
This `#variant` thing is cool, is that ActiveStorage?

Reply:
It is. It transforms the original image. In this case, making a standard thumbnail size.